control-plane/connect-inject/handler.go

```diff
@@ -138,6 +138,10 @@ type Handler struct {
 	// from mesh services.
 	EnableConsulDNS bool
 
+	// ReleasePrefix is the prefix used for the installation which is used to determine the Service
+	// name of the Consul DNS service.
+	ReleasePrefix string
```

Comment:
Can you use `ResourcePrefix` to match server-acl-init, create-federation-job and others